Airports/EGPK/Freetext.txt

```diff
@@ -0,0 +1,16 @@
+;EGPK SRA Advisory Altitudes
+N055.33.31.010:W004.44.15.626:EGPK SRA 12 altitudes:5-1600'
+N055.33.00.217:W004.42.42.973:EGPK SRA 12 altitudes:4-1290'
+N055.32.29.768:W004.41.16.613:EGPK SRA 12 altitudes:3-980'
+N055.31.56.898:W004.39.41.042:EGPK SRA 12 altitudes:2-660'
+N055.30.54.962:W004.36.40.262:EGPK SRA 12 altitudes:THR-38'
+N055.34.47.405:W004.30.59.963:EGPK SRA 21 altitudes:5-2000'
+N055.33.54.466:W004.31.41.416:EGPK SRA 21 altitudes:4-1610'
+N055.32.58.963:W004.32.24.061:EGPK SRA 21 altitudes:3-1230'
```

Comment:
```suggestion
N055.32.58.963:W004.32.24.061:EGPK SRA 21 altitudes:3-1220'
```
This is 1220 according to [this](https://www.aurora.nats.co.uk/htmlAIP/Publications/2021-04-22-AIRAC/graphics/101273.pdf)